app/src/main/java/org/sil/gatherwords/MainActivity.java

```diff
@@ -62,14 +85,9 @@ public View getView(int i, View convertView, ViewGroup container) {
                 );
             }
 
+            // get the date to prove that there is data being retrieved
             TextView dateText = convertView.findViewById(R.id.date);
-            dateText.setText("1:42 PM 1/5/2018");
-
-            TextView locationText = convertView.findViewById(R.id.location);
-            locationText.setText("Thailand");
```

Comment:
The entity has the data, why not fill this?

Reply:
The individual items in the list are more of proving that things can be read from the database. These aren't in the database.